assessment_store/db/models/flags/flag_update.py

```diff
@@ -26,7 +26,7 @@ class FlagUpdate(BaseModel):
         UUID(as_uuid=True),
         ForeignKey("assessment_flag.id"),
     )
-    user_id = Column("user_id", String, nullable=False)
+    user_id = Column("user_id", String, nullable=True)
```

Comment:
Can this be nullable?

Reply:
Originally a user had to resolve a flag, but now in `submit_application`, we automatically resolve flags that are change requests and have been addressed. So there's no "user" to specify in that situation.

Reply:
Is the user in this case maybe the person submitting the application?

Reply:
Good point! Completely missed that